core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html

```diff
@@ -227,4 +288,67 @@
     color: #fff;
     padding: 1px 5px;
   }
+  .oppia-contributor-home {
+    position: -webkit-sticky;
+    position: sticky;
+    top: 56px;
+    z-index: 3;
+  }
+  .headroom--unpinned {
+    transform: translateY(0%);
+  }
+  .collapsed-oppia-contributor-username-container {
+    color: #009688;
+    font-family: "Capriola", "Roboto", Arial, sans-serif;
+    font-size: 15px;
+    height: 30%;
+    position: relative;
+    width: 10%;
+  }
+  .collapsed-oppia-contributor-img {
+    display: block;
+    height: 32px;
+    margin: auto;
+  }
+  .collapsed-oppia-opportunities-tabs-explanation {
+    color: #009688;
+    font-family: "Capriola", "Roboto", Arial, sans-serif;
+    font-size: 15px;
+    position: relative;
+    width: 500px
+  }
+  .collapsed-oppia-opportunities-tab-custmization-container {
```

Comment:
Can you fix the typo in `custmization` while we're at it?

Reply:
Done